Code/Drivers/CGbasicSolver.c

```diff
@@ -65,11 +68,19 @@ int main(int argc, char *argv[])
     cgretcsr_p = mpiCGsolveCSR(pmdat,A_CSR,b_p_csr,xguess,tol);
     cgretdia_p = mpiCGsolveDIA(pmdat,A_DIA,b_p_csr,xguess,tol);
 
+    //Jacobi PC

```

Comment:
Clarifying question: We can add more blocks here with the appropriate strings to switch which type of preconditioner we are using, correct?